### ISSUE TITLE
A drag-and-drop operation for adding an entity to the scene has an error

### ISSUE THREAD

**Issue by EternalTamago** [SiliconStudio/xenko#335]
1. Create a new project (NewGame).
2. Create a procedual model (or import a 3D model).
3. Add the model to the entity list of the scene pane using a drag-and-drop operation.

![image](https://cloud.githubusercontent.com/assets/13661631/11182778/c4cc36b4-8cb0-11e5-8f64-5b9e0d0e8058.png)

```
Paradox Version: 1.3.4-beta
GameStudio Version: 4
Default session graphics profile: Level_10_0
Current Directory: D:\Programs\Silicon Studio\Paradox
Command Line Args: 
OS Version: Microsoft Windows NT 6.2.9200.0 (x64)
Processor Count: 4
Video configuration:
GPU 1
BitsPerPel: 32
Caption: NVIDIA GeForce GTX 660  
Description: NVIDIA GeForce GTX 660  
DeviceName: NVIDIA GeForce GTX 660  
DisplayFlags: 0
DisplayFrequency: 60
DitherType: 
DriverVersion: 
ICMIntent: 
ICMMethod: 
LogPixels: 96
PelsHeight: 768
PelsWidth: 1024
SettingID: NVIDIA GeForce GTX 660  
SpecificationVersion: 1025
Exception: KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at SiliconStudio.Paradox.Assets.Presentation.SceneEditor.SceneLoader.UnregisterReferencedObject(Guid id)
   at SiliconStudio.Paradox.Assets.Presentation.ViewModel.SceneObjectViewModel.RemoveSubEntities(IEnumerable subEntitiesToRemove)
   at SiliconStudio.Paradox.Assets.Presentation.ViewModel.SceneObjectViewModel.SubEntityCollectionChanged(Object sender, NotifyCollectionChangedEventArgs e)
   at System.Collections.Specialized.NotifyCollectionChangedEventHandler.Invoke(Object sender, NotifyCollectionChangedEventArgs e)
   at SiliconStudio.Presentation.Collections.ObservableList`1.OnCollectionChanged(NotifyCollectionChangedEventArgs arg)
   at SiliconStudio.Presentation.Collections.ObservableList`1.RemoveAt(Int32 index)
   at SiliconStudio.Presentation.Collections.ObservableList`1.Remove(T item)
   at SiliconStudio.Paradox.Assets.Presentation.ViewModel.SceneObjectViewModel.SiliconStudio.Assets.Editor.ViewModel.IInsertChildViewModel.InsertChildren(IReadOnlyCollection`1 children, InsertPosition position)
   at SiliconStudio.Assets.Editor.View.Behaviors.DragDropBehavior`2.OnDrop(Object sender, DragEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.OleDropTarget.RaiseDragEvent(RoutedEvent dragEvent, Int32 dragDropKeyStates, Int32& effects, DependencyObject target, Point targetPoint)
   at System.Windows.OleDropTarget.MS.Win32.UnsafeNativeMethods.IOleDropTarget.OleDrop(Object data, Int32 dragDropKeyStates, Int64 point, Int32& effects)
   at MS.Win32.UnsafeNativeMethods.DoDragDrop(IDataObject dataObject, IOleDropSource dropSource, Int32 allowedEffects, Int32[] finalEffect)
   at System.Windows.OleServicesContext.OleDoDragDrop(IDataObject dataObject, IOleDropSource dropSource, Int32 allowedEffects, Int32[] finalEffect)
   at System.Windows.DragDrop.OleDoDragDrop(DependencyObject dragSource, DataObject dataObject, DragDropEffects allowedEffects)
   at System.Windows.DragDrop.DoDragDrop(DependencyObject dragSource, Object data, DragDropEffects allowedEffects)
   at SiliconStudio.Assets.Editor.View.Behaviors.DragDropBehavior`2.PreviewMouseMove(Object sender, MouseEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at SiliconStudio.Paradox.GameStudio.Program.Main()


```

**Comment by Benlitz** [comment]
Thank you the report,

The problem has been fixed and will be integrated in the next release

**Comment by EternalTamago** [comment]
This is fixed in the Xenko Game Studio 1.4.1-beta! Thank you!
